src/lightning_app/cli/lightning_cli.py

```diff
@@ -49,10 +51,89 @@ def main():
 @click.group()
 @click.version_option(ver)
 def _main():
-    register_all_external_components()
+    # register_all_external_components()
     pass
 
 
+@_main.group()
+def show():
+    """Show given resource."""
+    pass
+
+
+@show.command()
+@click.argument("app_name", required=False)
+@click.argument("components", nargs=-1, required=False)
+@click.option("-f", "--follow", required=False, is_flag=True, help="Wait for new logs, to exit use CTRL+C.")
+def logs(app_name: str, components: List[str], follow: bool) -> None:
+    """Show application logs. By default prints logs for all currently available components.
+
+    Example uses:
+
+        Print all application logs:
+
+            $ lightning show logs my-application
+
+
+        Print logs only from the root flow component:
+
+            $ lightning show logs my-application flow
+
+
+        Print logs only from selected components:
+
+            $ lightning show logs my-application root.comp_a root.comp_b
+    """
+
+    client = LightningClient()
+    project = _get_project(client)
+
+    apps = {
+        app.name: app
+        for app in client.lightningapp_instance_service_list_lightningapp_instances(project.project_id).lightningapps
+    }
+
+    if not apps:
+        raise click.ClickException("Your app list is empty. Please run an application first.")
```

Comment:
Does this include stopped apps or only running ones?

Reply:
non-deleted ones. So getting logs from stopped will be ok